lua/lazyvim/plugins/extras/lang/zig.lua

```diff
@@ -0,0 +1,68 @@
+return {
+  {
+    "NTBBloodbath/zig-tools.nvim",
+    ft = "zig",
+    config = function()
```

Comment:
I suggest you use `opts = {}` or `config = true`. This way the user can pass his own changes to the `opts`, without any need to fiddle with the `config` function. Also, by using `config` like this, you're overriding the default `config` function and since you don't pass any `opts` to the new `config` function, the user would have to pass all the settings in the `setup` table of the `config` function in his own configuration.

Reply:
I have made the change for opts [49803c17accb133ac29d9357f1bf46512b3b65c2](https://github.com/LazyVim/LazyVim/pull/1750/commits/49803c17accb133ac29d9357f1bf46512b3b65c2)